contrib_bots/lib/translate.py

```diff
@@ -0,0 +1,95 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+from __future__ import print_function
+import sys
+import logging
+try:
+    from mstranslator import Translator
+except ImportError:
+    logging.error("Dependency Missing!")
+    sys.exit(1)
+
+#Uses mstranslator module
+#  pip install mstranslator
+
+translator = Translator('8f71c4d183fe45cca6fd91cdf73f12ea')
```

Comment:
Be very careful with not adding API keys or other credentials in any public code. You should remove this, and add some option for people using the bot to include their own API key.

This can be done with hardcoded strings, but preferrably use an external configuration file.

Also, add some brief explanation in a comment, on how to get that API key.